migrations/versions/846048f2921b_add_privacy_settings_to_user.py

```diff
@@ -0,0 +1,62 @@
+"""Add privacy settings to User.
+
+Revision ID: 846048f2921b
+Revises: 0309db1e86c0
+Create Date: 2025-08-26 17:09:39.228021
+
+"""
+
+from alembic import op
+import sqlalchemy as sa
+
+
+# revision identifiers, used by Alembic.
+revision = "846048f2921b"
+down_revision = "0309db1e86c0"
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    # ### commands auto generated by Alembic - please adjust! ###
+    #
+    with op.batch_alter_table("user", schema=None) as batch_op:
+        batch_op.add_column(
+            sa.Column("accepted_privacy_policy", sa.Boolean(), nullable=True)
+        )
+        batch_op.add_column(
+            sa.Column("do_individual_outreach", sa.Boolean(), nullable=True)
+        )
+        batch_op.add_column(sa.Column("send_newsletter", sa.Boolean(), nullable=True))
+
+    # manually added data migration - just set all the new columns to False.
+    with op.batch_alter_table("user", schema=None) as batch_op:
```

Comment:
Had to pull this out into a separate context manager from the column-adding operations to get these ops to see the new columns. In retrospect I could have probably done some sort of "commit this transaction, for real" thing in the first block, but this seems fine :shrug:

Reply:
why is `server_default` not appropriate here?

Reply:
(I mean this works fine but)